tests/containers/helm.pm

```diff
@@ -24,6 +24,10 @@ use registration qw(add_suseconnect_product get_addon_fullname);
 use containers::k8s;
 use publiccloud::utils qw(gcloud_install);
 
+my $chart_repo_url = "https://helm.nginx.com/stable";
+my $chart_repo = "nginx-stable";
+my $chart_name = "nginx-ingress";
```

Comment:
Why have these as global variables?

Reply:
These are just constant. I don't see a reason to have them sitting inside of the subroutine..

Reply:
> These are just constants. I don't see a reason to have them sitting inside of the subroutine..

Unfortunately they're variables in Perl.  And even if they're constants, it's good programming practice to limit their scope.

Reply:
Afaik it's still scoped only to that perl module, or am I mistaken?

Reply:
> Afaik it's still scoped only to that perl module, or am I mistaken?

Yet, they're global to the whole module.  We avoid global variables like the plague.

Reply:
Yes. While strictly speaking a style issue, there is some form of unspoken convention that such small and static things don't become variables because they don't change and are only used in 1-2 places.

I would simply replace the link and chart names and not make them variables, because there is little gain in doing so here.

Reply:
Reverted to the previous design.